code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -18,8 +18,6 @@
 /datum/symptom/heal/Start(datum/disease/advance/A)
 	if(!..())
 		return FALSE
-	if(A.stage_rate >= 6) //stronger healing
-		power = 2
 	return TRUE //For super calls of subclasses
```

Comment:
Threshold descriptions are now incorrect. You may be fixing an exploit, but you are simultanously fixing something completely intended.

Reply:
alright, ill remove the threshold description from the unused healing symptom, then